Cant Compile on NixOS

When i try to Compile on (NixOS with wayland) i get thus build error:


```
error: The platform you're compiling for is not supported by winit
  --> /home/ver/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.30.5/src/platform_impl/mod.rs:78:1
   |
78 | compile_error!("The platform you're compiling for is not supported by winit");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0432]: unresolved import `self::platform`
  --> /home/ver/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.30.5/src/platform_impl/mod.rs:34:15
   |
34 | pub use self::platform::*;
   |               ^^^^^^^^ could not find `platform` in `self`

error[E0432]: unresolved imports `crate::platform_impl::PlatformCustomCursor`, `crate::platform_impl::PlatformCustomCursorSource`
 --> /home/ver/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.30.5/src/cursor.rs:8:28
  |
8 | use crate::platform_impl::{PlatformCustomCursor, PlatformCustomCursorSource};
  |                            ^^^^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^^^^^^^^^^ no `PlatformCustomCursorSource` in `platform_impl`
  |                            |
  |                            no `PlatformCustomCursor` in `platform_impl`

error[E0432]: unresolved import `crate::platform_impl::PlatformIcon`
 --> /home/ver/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.30.5/src/icon.rs:1:5
  |
1 | use crate::platform_impl::PlatformIcon;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `PlatformIcon` in `platform_impl`

error[E0432]: unresolved import `crate::platform_impl::PlatformSpecificWindowAttributes`
 --> /home/ver/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.30.5/src/window.rs:7:34
  |
7 | use crate::platform_impl::{self, PlatformSpecificWindowAttributes};
  |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `PlatformSpecificWindowAttributes` in `platform_impl`

   Compiling re_memory v0.19.0
error[E0369]: binary operation `==` cannot be applied to type `&Option<MonitorHandle>`
    --> /home/ver/.cargo/registry/src/index.crates.io-6f17d22bba15001f/winit-0.30.5/src/window.rs:1750:16
     |
1745 | #[derive(Clone, Debug, PartialEq, Eq)]
     |                        --------- in this derive macro expansion
...
1750 |     Borderless(Option<MonitorHandle>),
     |                ^^^^^^^^^^^^^^^^^^^^^
     |
     = note: this error originates in the derive macro `PartialEq` (in Nightly builds, run with -Z macro-backtrace for more info)
     ```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -74,6 +74,8 @@ egui = { version = "0.29.0" }
 eframe = { version = "0.29.0", default-features = false, features = [
     "wgpu",
     "android-game-activity",
+    "wayland",
+    "x11",
 ] }
 egui_tiles.version = "0.10.1"
 

